lib/lograge/sql.rb

```diff
@@ -11,11 +11,14 @@ class << self
       attr_accessor :formatter
       # Extract information from SQL event
       attr_accessor :extract_event
+      # Filter SQL events by duration
+      attr_accessor :limit_duration
```

Comment:
Please rename this to `min_duration_ms`.
"limit" normally refers to an upper limit (max) so it could be confusing for people.
The `_ms` suffix also makes it easy to know this is supposed to be in ms, without the need to read the documentation.

Reply:
@iMacTia 

I agreed. Thank you 🙏